config/routes.rb

```diff
@@ -2,5 +2,11 @@
   get 'travel_time/get'
   resources :position_imports, only: [:create]
 
+  scope 'api' do
+    resources :rehire_sites do
```

Comment:
This should be restricted to the available actions for rehire_sites. See [routing concerns](http://guides.rubyonrails.org/routing.html#routing-concerns) for the syntax of using "only" with resources.